model-optimizer/extensions/front/ExpandDimsToUnsqueeze.py

```diff
@@ -35,5 +35,10 @@ def find_and_replace_pattern(self, graph: Graph):
                 expand_dims_node.in_port(0).get_connection().set_destination(unsqueeze_node.in_port(0))
                 expand_dims_node.out_port(0).get_connection().set_source(unsqueeze_node.out_port(0))
                 unsqueeze_node.in_port(1).connect(unsqueeze_dims_node.out_port(0))
+            elif len(expand_dims_node.in_nodes()) == 2:
+                unsqueeze_node = Unsqueeze(graph, {'name': expand_dims_node.id + '/Unsqueeze'}).create_node()
```

Comment:
here is the right code to retrieve the node name and how to preserve the node name (since the old one will be removed, we need to transfer its name to the new node)

```
expand_dims_name = expand_dims_node.soft_get('name', expand_dims_node.id)
unsqueeze_node = Unsqueeze(graph, {'name': expand_dims_name  + '/Unsqueeze'}).create_node()
rename_nodes([(expand_dims_node, expand_dims_name  + "/TBR"), (unsqueeze_node, expand_dims_name)])
```